src/unistd.rs

```diff
@@ -111,6 +114,92 @@ pub fn chdir<P: ?Sized + NixPath>(path: &P) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// Creates new directory `path` with access rights `mode`.
+///
+/// # Errors
+///
+/// `Err` is returned in case of an error. There are several situations where mkdir might fail.
+/// For a full list consult `man mkdir(2)`
```

Comment:
I think I would prefer a link to the man pages from man7.org, in this case http://man7.org/linux/man-pages/man2/mkdir.2.html